### PR TITLE
Add a keybind to enable/disable parcool globally

### PR DESCRIPTION
**Pull request by Kasualix**
Tested and works well.

Mainly for my own private modpack usage but you can merge this if you like :)

https://github.com/alRex-U/ParCool/assets/110756250/234d8dea-8e3b-4f3c-a4fb-3c85bb419bf5

### CODE REVIEW

File: src/main/java/com/alrex/parcool/client/input/KeyRecorder.java [hunk 1]
```diff
@@ -29,6 +34,15 @@ public class KeyRecorder {
 	public static void onClientTick(TickEvent.ClientTickEvent event) {
 		if (event.phase != TickEvent.Phase.START) return;
 
+		if (KeyBindings.getKeyBindEnable().consumeClick()) {
```
Comment (alRex-U) [inline src/main/java/com/alrex/parcool/client/input/KeyRecorder.java:37]: I think only recording of key status should be here and actual processing should be at new event consumer like these  ([com.alrex.parcool.common.event](https://github.com/alRex-U/ParCool/tree/main/src/main/java/com/alrex/parcool/common/event)). How do you think?

**Review by alRex-U — COMMENTED** [review]
I have one question about processing of enable key. Should processing it be done by class KeyRecorder?

File: src/main/java/com/alrex/parcool/client/input/KeyRecorder.java (hunk 1 above)
Comment (Kasualix) [reply src/main/java/com/alrex/parcool/client/input/KeyRecorder.java]: Migrated. tested & works fine too.

**Comment by alRex-U** [comment]
OK, Thank you for contributing.

**Pull request merged**